app/src/app/campaigns-grid/CampaignsGrid.tsx

```diff
@@ -0,0 +1,45 @@
+import clsx from "clsx";
+import styles from "./CampaignsGrid.module.scss";
+
+import { Grid } from "ui/grid/Grid";
+import { Button } from "ui/button/Button";
+import { Container } from "react-grid-system";
+import { Typography } from "ui/typography/Typography";
+import { useAuthContext } from "hooks/useAuthContext/useAuthContext";
+import { CampaignsGridProps } from "./CampaignsGrid.types";
+import { CampaignsGroup } from "./CampaignsGroup/CampaignsGroup";
+
+export const CampaignsGrid: React.FC<CampaignsGridProps> = ({ campaigns, className }) => {
+  const auth = useAuthContext();
+
+  return (
+    <div className={clsx(styles["campaigns-grid"], className)}>
+      {!auth.hasActiveSession && (
+        <section id="intro" className={styles["campaigns-grid__section"]}>
+          <Container>
+            <Grid.Row>
+              <Grid.Col>
+                <Typography.Headline1>Explora e Invierte con Bitcoin</Typography.Headline1>
+                <Typography.TextLead>Diversifíca tu portafolio invirtiendo en negocios locales</Typography.TextLead>
```

Comment:
Este tipo de líneas hay que discutirlas, para definir cómo nos dirigimos a la audiencia.

```suggestion
                <Typography.TextLead>Diversifica tu portafolio con ingresos pasivos desde 1 SAT.</Typography.TextLead>
```

Reply:
De acuerdo :)